src/parameterizations/lateral/MOM_thickness_diffuse.F90

```diff
@@ -804,8 +805,8 @@ subroutine thickness_diffuse_full(h, e, Kh_u, Kh_v, tv, uhD, vhD, cg1, dt, G, GV
         call calculate_density_derivs(T_u, S_u, pres_u, drho_dT_u, drho_dS_u, &
                                       tv%eqn_of_state, EOSdom_u)
       endif
-      if (use_Stanley) then
-        do i=is-1,ie
+      if (use_stanley) then
+        do i=is-1,ie+1
```

Comment:
I assume the `+1` here and the `+2` becoming ~+3` on line 818 are the bug you mentioned in the meeting?

Reply:
Yes, correct.